build/verify-archive.sh

```diff
@@ -10,8 +10,9 @@ set -euo pipefail
 apt-get update
 apt-get install -y cmake xz-utils
 
-tar xzf cockroach.src.tgz
-(cd cockroach-* && make install)
+workdir="$(mktemp -d)"
```

Comment:
my shellcheck-fu is a little rust, but I believe you don't need the quotes here.

probably best to consult shellcheck, though.